backend/backend.go

```diff
@@ -555,6 +557,38 @@ func (backend *Backend) Coin(code coinpkg.Code) (coinpkg.Coin, error) {
 	return coin, nil
 }
 
+func (backend *Backend) pollETHBalances() {
+	timer := time.After(0)
+
+	for {
+		<-timer
+		if err := backend.updateETHAccounts(); err != nil {
+			backend.log.WithError(err).Error("could not update ETH accounts")
+		}
+		timer = time.After(eth.PollInterval)
+	}
+}
+
+func (backend *Backend) updateETHAccounts() error {
+	defer backend.accountsAndKeystoreLock.RLock()()
+	backend.log.Debug("Updating ETH accounts balances")
+
+	ethAccounts := make([]*eth.Account, 0, len(backend.accounts))
+	for _, account := range backend.accounts {
+		ethAccount, ok := account.(*eth.Account)
+		if ok {
+			ethAccounts = append(ethAccounts, ethAccount)
+		}
+
+	}
+
+	etherScanClient := etherscan.NewEtherScan("1", backend.etherScanHTTPClient)
```

Comment:
Was wondering about the rate limiting, and now see that https://github.com/BitBoxSwiss/bitbox-wallet-app/commit/e509f1838ab5a81f571fe843919eb3e1ee1a755f#diff-ceb7e04e1612df83dc6c0f93e8dd3646a846c5949195088be13031f700264b67 introduced a bug:

Before, the rate limiting was enforced across all instances of the etherscan client, and now it's per-instance. This means that us making a separate instance per ERC-20 token and making new instances here goes over our intended rate limit.

Should probably instantiate the `rate.Limiter` instance once and pass it to all instances.